From 4f4bb63cdb4b11fa1c0a4f48f7042e0e54222eb5 Mon Sep 17 00:00:00 2001
From: Shashank A Patil <90090313+shashstormer@users.noreply.github.com>
Date: Tue, 2 Jan 2024 00:11:00 +0530
Subject: [PATCH] fixed quick redirection regex __init__.py

a solution was proposed multiple days ago but no one was commiting it to the repo so created this commit based on https://github.com/justfoolingaround/animdl/issues/295
---
 animdl/core/codebase/providers/animepahe/inner/__init__.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/animdl/core/codebase/providers/animepahe/inner/__init__.py b/animdl/core/codebase/providers/animepahe/inner/__init__.py
index 357e7c58..1dd5a580 100644
--- animdl/core/codebase/providers/animepahe/inner/__init__.py
+++ animdl/core/codebase/providers/animepahe/inner/__init__.py
@@ -4,7 +4,7 @@
 KWIK_D_URL = regex.compile(r'action="(.+?)"')
 KWIK_D_TOKEN = regex.compile(r'value="(.+?)"')
 
-KWIK_REDIRECTION_RE = regex.compile(r'<a href="(.+?)" .+?>Redirect me</a>')
+KWIK_REDIRECTION_RE = regex.compile(r';\$\("a\.redirect"\)\.attr\("href","(.+?)"\)\.html\("Continue"\);')
 
 CHARACTER_MAP = "0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ+/"
 
